website/docs/golioth-exploration/02-device-interaction/01-device-status.md

```diff
@@ -27,7 +27,7 @@ On this page you can view the following information:
 * `Device Id`: Used by the Golioth API to identify this device
 * `History`: Displays when this virtual device was created and last updated
 * `Status`: Displays information on when device last checked in and if its
-  Settings Service has confirmed lasted cloud-side settings
+  Settings Service has confirmed ~~lasted~~ cloud-side settings
```

Comment:
"lasted" doesn't seem like the right word here, but I'm not sure what it should say instead.